src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -312,18 +338,18 @@ String ^ UnitConverterViewModel::ConvertToLocalizedString(const std::wstring& st
                 // typed a post-decimal digit.
 
                 m_decimalFormatter->IsDecimalPointAlwaysDisplayed = true;
-                m_currencyFormatter->IsDecimalPointAlwaysDisplayed = true;
+                currencyFormatter->IsDecimalPointAlwaysDisplayed = true;
             }
 
             // force post-decimal digits so that trailing zeroes entered by the user aren't suddenly cut off.
             m_decimalFormatter->FractionDigits = static_cast<int>(stringToLocalize.length() - (posOfDecimal + 1));
-            m_currencyFormatter->FractionDigits = m_currencyMaxFractionDigits;
+            currencyFormatter->FractionDigits = lastCurrencyFractionDigits;
         }
 
         if (IsCurrencyCurrentCategory)
         {
-            wstring currencyResult = m_currencyFormatter->Format(stod(stringToLocalize))->Data();
-            wstring currencyCode = m_currencyFormatter->Currency->Data();
+            wstring currencyResult = currencyFormatter->Format(stod(stringToLocalize))->Data();
```

Comment:
This results in the conversions to Yen from other currencies always displaying a decimal, is this existing behavior?

![image](https://user-images.githubusercontent.com/6334170/99326875-8ed69900-282d-11eb-8333-21a312be38ac.png)

Reply:
It's an existing behavior. If you change locale to Japanese, all currency converted to will have a decimal:
![image](https://user-images.githubusercontent.com/2201482/99337045-85484200-28bc-11eb-82dd-0590ea5c488d.png)

But I think we can discuss about this -- is it really needed?

Reply:
I don't think it's needed, would be great if it is easy to remove!

Reply:
Should be easy. Let me do it together with disabling decimal input.